package.json

```diff
@@ -20,6 +20,8 @@
   },
   "scripts": {
     "clean": "rimraf lib",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "check-format": "prettier --list-different \"src/**/*.ts\"",
```

Comment:
I think we should have the build fail if there is formatting issues (similar to lint failures). Could this get added to the build command?

Reply:
Yep. I'll add